resources/lang/de/injuries.php

```diff
@@ -11,91 +11,100 @@
-    'Dislocated Shoulder'       => 'Dislocated Shoulder',
-    'Broken Leg'                => 'Broken Leg',
-    'Gluteal Strain'            => 'Gluteal Strain',
-    'Achilles tendonitis'       => 'Achilles tendonitis',
-    'Metatarsal Fracture'       => 'Metatarsal Fracture',
-    'Heel Injury'               => 'Heel Injury',
-    'Gashed Leg'                => 'Gashed Leg',
-    'Broken Hand'               => 'Broken Hand',
-    'Appendicitis'              => 'Appendicitis',
-    'Broken Arm'                => 'Broken Arm',
-    'Eye Injury'                => 'Eye Injury',
-    'Hip Labrum Tear'           => 'Hip Labrum Tear',
-    'Patella Fracture'          => 'Patella Fracture',
-    'LCL Knee Ligament Injury'  => 'LCL Knee Ligament Injury',
-    'Viral Meningitis'          => 'Viral Meningitis',
-    'Medical Condition'         => 'Medical Condition',
-    'Wisdom Teeth'              => 'Wisdom Teeth',
-    'Chin Injury'               => 'Chin Injury',
-    'Fractured Cheekbone'       => 'Fractured Cheekbone',
-    'Kidney Injury'             => 'Kidney Injury',
-    'Broken Toe'                => 'Broken Toe',
-    'Fibula fracture'           => 'Fibula fracture',
-    'Twisted Knee'              => 'Twisted Knee',
-    'Broken Nose'               => 'Broken Nose',
-    'Broken Collarbone'         => 'Broken Collarbone',
-    'Bowel Disease'             => 'Bowel Disease',
-    'Sciatica'                  => 'Sciatica',
-    'Testicular Cancer'         => 'Testicular Cancer',
-    'Broken Finger'             => 'Broken Finger',
-    'Rib Fracture'              => 'Rib Fracture',
-    'Ankle Fracture'            => 'Ankle Fracture',
-    'Broken Shinbone'           => 'Broken Shinbone',
-    'Broken Wrist'              => 'Broken Wrist',
-    'Hand Injury'               => 'Hand Injury',
-    'Ear Infection'             => 'Ear Infection',
+    'Unknown'                   => 'Unbekannt',
+    'Ankle/Foot Injury'         => 'Knöchelverletzung',
+    'Facial Injury'             => 'Gesichtsverletzung',
+    /*
+     * Note: I'm not sure, which tendon you mean, as the word is ambiguos.
+     * Tendon in the knee: "Kreuzbandriss" (pretty common injury in the knee, a rupture of the Ligamenta cruciata genus)
+     * Tendon at the back of the foot: "Achillessehne" (the achilles tendon, rupturing that would render a person unable to walk)
+     * Injury of tendons in general: Sehnenzerrung
+     */
+    'Hamstring'                 => 'Sehnenzerrung',
+    'Calf/Shin Injury'          => 'Schienbeinverletzung',
+    'Hip/Thigh Injury'          => 'Hüftverletzung',
+    'Suspended'                 => 'Gesperrt',
+    'Thigh Muscle Strain'       => 'Oberschenkelmuskelzerrung',
+    'MCL Knee Ligament Injury'  => 'Verletzung am Außenband (Knie)',
+    'Groin Strain'              => 'Leistenzerrung',
+    /*
+     * Note: How does a player get the condition "Knock"? Again, the literal translation is likely to be wrong.
```

Comment:
Note can be removed. I need to ask the API-supplier what this injuries means

Reply:
All right :+1: